code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -524,11 +524,13 @@ Basically, we fill the time between now and 2s from now with hands based off the
 	ADD_TRAIT(owner, TRAIT_NOSOFTCRIT, type)
 	ADD_TRAIT(owner, TRAIT_NOCRITDAMAGE, type)
 	ADD_TRAIT(owner, TRAIT_NODEATH, type)
+	ADD_TRAIT(owner, TRAIT_NOCRITOVERLAY, type)
```

Comment:
Why not make this inherent to NOHARDCRIT? If it's very debilitating to nooartrium users it's propably also debilitating to other NOHARDCRIT users

Reply:
This is because the crit overlay acts as an indicator of status for penthrite users.

Penthrite users can't reach -200 health where the overlay actually becomes blinding, I've used penthrite before and honestly was glad that I could tell how close the ash drake (that was hugging me as a golem) was to literally gibbing me.

Reply:
Fair enough